File: web/src/interfaces/PessoaDtos/UpdatePessoaDto.ts
export class UpdatePessoaDto {
  id: string = '';
  nome: string = "";
  idade: number = 0;
  cpf: string = "";
  dataCriacao: string = new Date().toISOString();

  constructor(
    id: string = '',
    nome: string = "",
    cpf: string = "",
    idade: number = 0,
    dataCriacao: string = new Date().toISOString(),
  ) {
    this.id = id;
    this.nome = nome;
    this.idade = idade;
    this.cpf = cpf;
    this.dataCriacao = dataCriacao
  }
}

File: api/HomeBudget.API/HomeBudget.Application/DTOs/PessoaDtos/UpdatePessoaDto.cs
namespace HomeBudget.Application.DTOs.PessoaDtos
{
    public class UpdatePessoaDto
    {
        public Guid Id { get; set; }
        public required string Nome { get; set; }
        public required string CPF { get; set; }
        public int Idade { get; set; }
        public DateTime DataCriacao { get; set; }
    }
}
